release-notes/sql-jdbc.release-notes-1.8.1.0.md

```diff
@@ -0,0 +1,4 @@
+## 2020-06-15, Version 1.8.1.0 (Current)
+### Changes
```

Comment:
Could you change it to "Features" instead

Reply:
Sure, changed